helm/wind-river-cloud-platform-deployment-manager/templates/crds.yaml

```diff
@@ -1907,6 +1907,12 @@ spec:
                 required:
                 - type
                 type: object
+              controller0Address:
```

Comment:
is this for duplex setup? i see scope is for simplex setup, how does C-0 and C-1 impact ?

Reply:
https://docs.starlingx.io/dist_cloud/kubernetes/update-a-subcloud-network-parameters-b76377641da4.html#aio-sx-subcloud

Under Step 2 of "AIO-SX subcloud" we have:

`~(keystone_admin)]$ system addrpool-add --floating-address 192.168.103.2 --controller0-address 192.168.103.3 --controller1-address 192.168.103.4 --gateway-address 192.168.103.1 admin 192.168.103.0 24`

The procedure says we need to provide all of floating address, c0 and c1 IPs, hence we have incorporated it accordingly.